### ISSUE TITLE
Python module missing dependency

### ISSUE THREAD

**Issue by vladyes** [LibreTime/libretime#950]
![Screenshot_2020-01-15-13-22-53-885_com server auditor ssh client](https://user-images.githubusercontent.com/59807725/72431308-8f357c00-37a5-11ea-8728-89f752dc0f88.jpg)
An error occurs while installing on Debian 9.
The installation is made on a digital machine. What could be the problem?

**Comment by kessibi** [comment]
I ran into the same problem with a Docker image installation.
Problem was solved by running this line: https://github.com/kessibi/libretime-docker/blob/0c5eb830cf7e2ae1ee87db98aa4814b3b7547514/scripts/preparation.sh#L17 before running the install script.

**Comment by paddatrapper** [comment]
Can you give more context? E.g. which Python application is giving the error and so missing the dependency

**Comment by vladyes** [comment]
> Я столкнулся с той же проблемой при установке образа Docker.
> Проблема была решена с помощью этой строки: https://github.com/kessibi/libretime-docker/blob/0c5eb830cf7e2ae1ee87db98aa4814b3b7547514/scripts/preparation.sh#L17 перед запуском сценария установки.



> I ran into the same problem with a Docker image installation.
> Problem was solved by running this line: https://github.com/kessibi/libretime-docker/blob/0c5eb830cf7e2ae1ee87db98aa4814b3b7547514/scripts/preparation.sh#L17 before running the install script.

Thanks, it helped me.
Installation completed successfully.

**Comment by kessibi** [comment]
I do not have the complete explanation as to why this package is needed but running `pip install more-itertools ` it beforehand makes it download the version 5.0.0 (from what I can recall) which is the better option for python __2.7__. Otherwise I believe it tried to install version 8.0.something which is designed for python __3__.

It's not a missing dependency per say but rather the wrong version you have to pre-install.

**Comment by codenift** [comment]
@kessibi yes. "`pip install more-itertools`" can be added to the **install** file. 

```
loud "\n-----------------------------------------------------"
loud "          * Installing Airtime Services *            "
loud "-----------------------------------------------------"

verbose "\n * Installing necessary python services..."
loudCmd "pip install setuptools --upgrade"
loudCmd "pip install more-itertools"
verbose "...Done"
```

It can be added it to line 908 right under "`loudCmd "pip install setuptools --upgrade`". That will take care of the issues.

**Comment by paddatrapper** [comment]
While that would work, a better solution would be to add a versioned dependency `more-itertools==5.0.0` to the `setup.py` of the python package that requires it.

Though, since #951 should fix this, a work around in the install script until that is merged should be fine

**Comment by codenift** [comment]
It seems like the only package needed was the zipp, but with a newer version it failed. We can add a previous version of that package instead. 

Instead of `more-itertools==5.0.0` use `zipp==1.0.0`

Tested and works.

**Comment by paddatrapper** [comment]
Fixed by #954
